File: campusconnect.client/src/app/entities/Dtos/moduleDto.ts
import { DifficultyEnum } from "../../enums/difficultyEnum";

export class ModuleDto {
    constructor(
        name: string, 
        facultyId: number, 
        difficulty: DifficultyEnum) {
            this.name = name;
            this.facultyId = facultyId;
            this.difficulty = difficulty;
    }

    public name: string;
    public facultyId: number;
    public difficulty: DifficultyEnum;
}

File: CampusConnect.Server/Models/Dtos/ModuleDto.cs
using CampusConnect.Server.Enums;
using System.Collections.Generic;

namespace CampusConnect.Server.Models.Dtos
{
    public class ModuleDto
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public string Difficulty { get; set; }
        public int FacultyId { get; set; }
    }
}
